src/components/FilterSelector.tsx

```diff
@@ -0,0 +1,56 @@
+import React, {MouseEvent} from 'react';
+import "../stylesheets/filterbuttons.scss"
+import {FilterSelectorEntry} from "./FilterSelectorEntry";
+import {FilterButton} from "./FilterButton";
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+
+export interface FilterSelectorState {
+    listOpen: boolean,
+    selected: string
+}
+
+export class FilterSelector extends React.Component<{defaultText: string},FilterSelectorState> {
```

Comment:
Suggestion: Maybe we can have as a FilterSelector prop, the list of selector items. If it's not too hard, that'll make it really convenient to reuse this component for other dropdowns.